src/EventBuilder/EventBuilder.csproj

```diff
@@ -1,130 +1,19 @@
-    <Optimize>true</Optimize>
-    <DebugType>pdbonly</DebugType>
-    <PlatformTarget>x86</PlatformTarget>
-    <ErrorReport>prompt</ErrorReport>
-    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
-    <Prefer32Bit>true</Prefer32Bit>
+    <AssemblyName>EventBuilder</AssemblyName>
+    <RootNamespace>EventBuilder</RootNamespace>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Cecil\DelegateTemplateInformation.cs" />
-    <Compile Include="Cecil\PathSearchAssemblyResolver.cs" />
-    <Compile Include="Cecil\EventTemplateInformation.cs" />
-    <Compile Include="Cecil\SafeTypes.cs" />
-    <Compile Include="CommandLineOptions.cs" />
-    <Compile Include="Entities\MultiParameterMethod.cs" />
-    <Compile Include="Entities\NamespaceInfo.cs" />
-    <Compile Include="Entities\ParentInfo.cs" />
-    <Compile Include="Entities\PublicEventInfo.cs" />
-    <Compile Include="Entities\PublicTypeInfo.cs" />
-    <Compile Include="Entities\SingleParameterMethod.cs" />
-    <Compile Include="PlatformHelper.cs" />
-    <Compile Include="Platforms\Android.cs" />
-    <Compile Include="Platforms\iOS.cs" />
-    <Compile Include="Platforms\IPlatform.cs" />
-    <Compile Include="Platforms\Mac.cs" />
-    <Compile Include="Platforms\Net45.cs" />
-    <Compile Include="Platforms\UWP.cs" />
-    <Compile Include="Platforms\XamForms.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Platforms\BasePlatform.cs" />
-    <Compile Include="Platforms\Bespoke.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="App.config" />
-    <None Include="Cecil\README.md" />
-    <None Include="DefaultTemplate.mustache">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="CommandLineParser">
-      <Version>1.9.71</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Web.Xdt">
-      <Version>2.1.1</Version>
-    </PackageReference>
-    <PackageReference Include="Mono.Cecil">
-      <Version>0.9.6.1</Version>
-    </PackageReference>
-    <PackageReference Include="NuGet.Core">
-      <Version>2.10.1</Version>
-    </PackageReference>
-    <PackageReference Include="Nustache">
-      <Version>1.15.3.7</Version>
-    </PackageReference>
-    <PackageReference Include="Polly">
-      <Version>3.0.0</Version>
-    </PackageReference>
-    <PackageReference Include="Serilog">
-      <Version>1.5.14</Version>
-    </PackageReference>
+
+<ItemGroup>
```

Comment:
Also, we might be able to pull the nuget restore into an MSBuild command and slightly simplify the cake file?